package.json

```diff
@@ -9,8 +9,9 @@
     ".": {
       "types": "./lib/index.d.ts",
       "require": "./lib/index.js",
-      "default": "./lib/index.mjs"
-    }
+      "default": "./lib/esm/index.js"
+    },
+    "./lib": "./lib/index.js"
```

Comment:
Shouldn't this also provide an object with `require` and `default` to point to `lib/index.js` and `lib/index.mjs` respectively?